addons/respawn/stringtable.xml

```diff
@@ -201,6 +201,7 @@
             <Russian>Удалять трупы игроков после дисконнекта?</Russian>
         </Key>
         <Key ID="STR_ACE_Respawn_Module_Description">
+            <English>This module allows you to setup respawn in your mission.</English>
```

Comment:
This is misleading. To setup respawn you have to use a combination of markers and/or respawn templates in description.ext. It's more like: "This module allows you to enable certain ACE functionality on respawn." (But please find better words, it's early in the morning atm :smile: )